app/events/event/route.js

```diff
@@ -1,10 +1,19 @@
 import Route from '@ember/routing/route';
+import { get } from '@ember/object';
+import { reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
 
 export default Route.extend({
+  fastboot: service(),
+  isFastBoot: reads('fastboot.isFastBoot'),
+
   titleToken(model) {
-    return model.get('title');
+    return get(model, 'title');
   },
   model({url_path}) {
+    if (this.get('isFastBoot')) {
+      return {'title': ''}
```

Comment:
At what point does this page title get populated?

Reply:
on load when the route is rehydrated